core/src/edu/asu/c3simulator/simulation/Company.java

```diff
@@ -0,0 +1,9 @@
+package edu.asu.c3simulator.simulation;
+
+import java.util.List;
+
+
```

Comment:
This interface looks good - it will help pull everything together when the company backend is created.

I don't see a javadoc though...